gwswlib/importer.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import logging
+import os
+import csv
+
+from gwswlib.hydx import Hydx, ConnectionNode
+
+logger = logging.getLogger(__name__)
+
+
+def importhydx(hydxpath):
+    """Read set of hydx-csvfiles and return Hydx objects"""
+    print("hoi")
+
+    hydx = Hydx()
+
+    # check if hydx exists
+    csvpath_knooppunt = os.path.join(hydxpath, "Knooppunt1.csv")
+    if not os.path.isfile(csvpath_knooppunt):
+        hint = "[!] ERROR : The specified file could not be found: "
+        print(hint, csvpath_knooppunt)
```

Comment:
Niet printen maar loggen (tenzij het tijdelijke debugzaken zijn, maar dan horen ze weer niet in een commit, tenzij het WIP is zoals nu, maar goed).